lib/chewy/fields/base.rb

```diff
@@ -53,6 +55,14 @@ def compose(*objects)
 
     private
 
+      def field_with_ignore_blank_flag?(result)
+        @options[:ignore_blank] == true && result.blank?
+      end
+
+      def geo_point_field_without_ignore_blank_flag?(result)
+        @options[:ignore_blank] != false && @options[:type] && @options[:type].to_sym == :geo_point && result.blank?
+      end
+
```

Comment:
```suggestion
   def geo_point?
     @options[:type].to_s == 'geo_point'
   end
   
   def ignore_blank?
     @options.fetch(:ignore_blank) { geo_point? }
   end
```

then you could use:
```ruby
return {} if result.blank? && ignore_blank?
```

Reply:
This would work only if you don't expect to ignore `ignore_blank` options for `geo_point` field.